testing/xmrig-proxy: new aport

OTLabs opened the pull request:
https://github.com/xmrig/xmrig-proxy
XMRig Proxy is a high performance Monero (XMR) Stratum proxy

andypost commented on testing/xmrig-proxy/APKBUILD:
```diff
@@ -0,0 +1,44 @@
+}
+
+package() {
+	msg2 'Packaging binary...'
```
this  and similar lines should be removed

andypost commented on testing/xmrig-proxy/APKBUILD:
```diff
@@ -0,0 +1,44 @@
+builddir="$srcdir/$pkgname-$pkgver"
+
+prepare() {
+	default_prepare
```
no reason in that

OTLabs commented:
Addressed all your recommendations @andypost Thank you!

OTLabs commented:
Update APKBUILD based on recommendations received for xmrig: removed dependencies and fixed install script.

libuv dependency was not traced as in xmrig. Should I add it manually?

andypost commented:
Commited ed291c9d
